src/canvas.js

```diff
@@ -217,12 +225,12 @@ class CanvasCard {
           });
 
       const headlineHeight =
-        (splitHeadline.length * Config.headline.fontSize[headlineSize] +
+        (splitHeadline.length * Config.headline.lineHeight[headlineSize] +
```

Comment:
what has prettier done to this line?!?! 😱

Reply:
I changed this as it was rendering part of the standfirst offscreen when the fontsize and line height differed.